Content.Server/Singularity/EntitySystems/SingularitySystem.cs

```diff
@@ -187,7 +209,10 @@ private void PullEntities(ServerSingularityComponent component, Vector2 worldPos
         /// </summary>
         private void DestroyTiles(ServerSingularityComponent component, Vector2 worldPos)
         {
-            var radius = DestroyTileRange(component);
+            // The - 0.05f here is a margin of error to stop a singularity from eating force field equipment.
+            // Raised to 0.05f because 0.01f wasn't enough (it's around certain internal physics margins).
+            // Note that it has to be a margin like this because if the singularity *envelopes* a target, it should still be eating it regardless.
+            var radius = DestroyTileRange(component) - 0.05f;
 
```

Comment:
You can't rely on this because due to how collision works it may overlap the fields if the station is moving even slightly.

Reply:
At that point I basically have to give up, I don't know of a good solution to this. I can't just ignore tiles that are on the fields because then the singularity might not eat fields it should be eating when put into a constrained situation (i.e. 3x3 box)

Reply:
Only thing I can think of is checking anchored entities (at least at lower energy levels) to ensure they aren't eaten. Given it will likely only be out of containment at higher energy levels it won't need to do the check anyway.